src/native/eventpipe/ep-types.h

```diff
@@ -188,6 +266,8 @@ struct _EventPipeProviderConfiguration_Internal {
 	const ep_char8_t *filter_data;
 	uint64_t keywords;
 	EventPipeEventLevel logging_level;
+	const EventPipeProviderEventFilter *event_filter;
```

Comment:
Since all "external" access to this should go through getters and since we converted it into a type with fini, we should remove const on these fields, including provider_name.

Reply:
Removed the const from the structs, and updated callsites to use getters if not in the corresponding ep*.c source.